pdfminer/layout.py

```diff
@@ -51,10 +51,15 @@ class LAParams:
         of a text matters when determining the order of text boxes. The value
         should be within the range of -1.0 (only horizontal position
         matters) to +1.0 (only vertical position matters).
+    :param cell_margin: (float) An additional distance allowing a line
+        to be counted as splitting an object when split_table mode enabled.
     :param detect_vertical: If vertical text should be considered during
         layout analysis
     :param all_texts: If layout analysis should be performed on text in
         figures.
+    :param split_tables: (bool) Enable parsing logic to have textboxes
+        and textlines to be bounded LTRect, LTLine, and LTCurve objects.
+        This causes the parser to respect boundary lines drawn a table.
```

Comment:
I'm not sure if we need this parameter. If this PR improves extraction of tables, don't we always want to use it?

Reply:
Great question.

I am worried that there are scenarios where users are interested in deactivating the splitting logic:  say that text is overlaid on some lines (say that the pdf was a map of some sort and you wanted to grab the name of a city).  I think that the user might want to turn off the splitting logic and let the boxes be made across lines.

I am wondering:  is there a better way to have the logic turned off?  I am worried that it can be implemented more elegantly.

Reply:
Agreed, there are cases where you want to switch this off/on. Any thoughts on what the default should be? 

You could use `cell_margin=None` to signal that it should not be used.

Reply:
I love your suggestion.  I have gone ahead and incorporated your suggestion.